### ISSUE TITLE
Research the possibility of and changes involved in building a 1.7.10 compatible version of the mod

### ISSUE THREAD

**Issue by pandapaul** [ToroCraft/Dailies#29]
A couple folks now have requested 1.7.10-compatible versions, so let's research it and determine how feasible it is.  Then we'll go from there.

**Comment by LEWeee** [comment]
I would love to see 1.7.10 version!

**Comment by pandapaul** [comment]
Yeah it's a popular request.  I definitely want to try to make it happen.  Just gonna take some time.  :)

**Comment by LEWeee** [comment]
I would like to help, but I'm just leanring the basics of Java, so I can't probably really help you much.
